server/config/config.go

```diff
@@ -969,6 +970,8 @@ func (man Manager) addConfigs() {
 		"Prefix URL for the CVE data feed. If empty, default to https://nvd.nist.gov/")
 	man.addConfigString("vulnerabilities.current_instance_checks", "auto",
 		"Allows to manually select an instance to do the vulnerability processing.")
+	man.addConfigBool("vulnerabilities.external_scheduled", false,
+		"Vulnerability processing is managed by an external 'cron' mechanism.")
```

Comment:
@zwass how strongly are you feeling about the configuration name? I don't like the double negative this ends up being in main.go, but it keeps the config a little bit more clear IMO.

Reply:
I don't feel very strongly, but I have a feeling about trying to make the flag descriptive of "what it does" (disable the job) rather than "when to use it" (when you are running the job separately).

Maybe it could be called `vulnerabilities.disable_schedule` and the description could say `Set this to true when the vulnerability processing job is scheduled by an external mechanism`?